storybook/src/components/Field/Field.docs.mdx

```diff
@@ -10,6 +10,14 @@ import README from "../../../../packages/css/src/components/field/README.md?raw"
 
 <Controls />
 
+## With Description
+
+A Field can have a description.
+Make sure to connect this description to the input in the Field,
+otherwise this won’t be read by a screen reader.
+
```

Comment:
Let’s be explicit in the markup people need to add, rather than letting them find it in the Code View.

```suggestion
Add an `aria-describedby` attribute to the Field
and provide the `id` of the describing element as its value.

```

Reply:
I see you’ve already did so with the examples for radio and checkbox.

Reply:
Your suggestion isn't quite correct, at least it isn't the current implementation.

```suggestion
Add an `aria-describedby` attribute to the input and provide the `id` of the describing element as its value.

```

Reply:
Sorry, badly copied and pasted from Field Set.